assets/bingo/lang/de_de.json

```diff
@@ -1,4 +1,9 @@
 {
+
+
+
+
+
```

Comment:
What's the padding for?

Reply:
For easier checking files side by side - with that I found [missing line](https://github.com/NeunEinser/bingo/pull/134/files#diff-1472095dcdb53ae3ffb36516720823c450b536f55248ef1a608d01ca3b97e37cR77) in [de_de]

Reply:
Ah so this is where technical strings are in en_us, so you have the same line numbers between both files?

Reply:
Yes, indeed ;)